recipes/qt/6.x.x/conanfile.py

```diff
@@ -235,7 +235,8 @@ def requirements(self):
             self.requires("pcre2/10.36")
         if self.options.get_safe("with_vulkan"):
             self.requires("vulkan-loader/1.2.172")
-
+            if self.settings.os == "Macos":
+                self.requires("moltenvk/1.1.1")
```

Comment:
How qt use MoltenVK? Does it link to MoltenVK? You probably don't want to link also vulkan-loader if MoltenVK is also linked.

Reply:
I think it is linked in runtime, as vulkan libs. But I am not 100% sure about it.
@ericLemanissier

Reply:
Do not resolve this conversation please.

Reply:
Have you tested if this PR works on Macos when vulkan is enabled?

Reply:
Ok, it seems that MoltenVK is linked to qt. So, if Apple OS, you have to require MoltenVK and NOT require vulkan-loader.